cli/src/rpc.rs

```diff
@@ -195,7 +197,8 @@ where
 	io.extend_with(CennzxApi::to_delegate(Cennzx::new(client.clone())));
 	io.extend_with(NftApi::to_delegate(Nft::new(client.clone())));
 	io.extend_with(StakingApi::to_delegate(Staking::new(client.clone())));
-	io.extend_with(GenericAssetApi::to_delegate(GenericAsset::new(client)));
+	io.extend_with(GenericAssetApi::to_delegate(GenericAsset::new(client.clone())));
+	io.extend_with(GovernanceApi::to_delegate(Governance::new(client)));
```

Comment:
do we need to clone here as well?
io.extend_with(GovernanceApi::to_delegate(Governance::new(client.clone())));

Reply:
they last one in the order gets to "move" the ownership so no clone needed